cfme/services/catalogs/catalog_item.py

```diff
@@ -170,7 +170,9 @@ def create(self):
             tabstrip.select_tab("Catalog")
             template = template_select_form.template_table.find_row_by_cells({
                 'Name': self.catalog_name,
-                'Provider': self.provider
+                # HACK to workaround BZ1390209
+                # revert back to Provider once it's fixed
```

Comment:
or I'll try it with something like `if BZ(1390209, forced_streams=["5.7", "upstream"]).blocks`...